src/rp/util/time.clj

```diff
@@ -10,7 +10,9 @@
 (def patterns
   {:review      "dd-MMM-yyyy"
    :floorplan   "MM/dd/yyyy"
-   :last-update "MM/dd/yyyy HH:mm:ss"})
+   :last-update "MM/dd/yyyy HH:mm:ss"
+   ;; Used for terminate_dm property in Rent Endeca
+   :terminate   "yyyyMMddHHmmss.SSSSSS"})
```

Comment:
If we don't mind a major version bump (to indicate a backward incompatible change), I could rename the formats such that the name is the format string itself... or perhaps even refactor away the use of named formats.
We only have a couple of places that depend on the named formats (in LS and rp-endeca).